mkxp.json

```diff
@@ -150,7 +150,7 @@
     // A list of fonts to render without alpha blending.
     // (default: none)
     //
-    // "solidFonts": ["Arial", "Times New Roman"]
+    // "solidFonts": ["Arial", "Times New Roman"],
```

Comment:
Is there any reason you didn't split this one up into multiple lines?

Reply:
It was already on one line. The splitting is mostly for the case where someone doesn't see that the first line doesn't have a closing bracket in the case of multiline arrays.

Reply:
Whatever, I'll split this line too for consistency